assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -225,9 +232,10 @@ const baseActions = {
 	 * @param {WPComponent} [settings.settingsEditComponent] Optional. React component to render the settings edit panel. Default none.
 	 * @param {WPComponent} [settings.settingsViewComponent] Optional. React component to render the settings view panel. Default none.
 	 * @param {WPComponent} [settings.setupComponent]        Optional. React component to render the setup panel. Default none.
-	 * @return {Object} Redux-style action.
+	 * @param {Function}    [settings.checkRequirements]     Optional. Function to check requirements for the module. Throws a WP error object for error or returns on success.
+	 * @return {void}
```

Comment:
`@return` should be omitted in case of `void`.

Reply:
Addressed.

Reply:
This needs to be added to #2355 as we currently have: 
```
     218:2  error  Missing JSDoc @return declaration    jsdoc/require-returns
     218:2  error  Missing JSDoc @returns for function  valid-jsdoc
```
Per https://github.com/google/site-kit-wp/blob/develop/.eslintrc.json#L31